react/tic-tac-toe/src/app/screens/Game/components/MatchHistory/index.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+
+import services from '~services/MatchesService'; // eslint-disable-line import/no-unresolved
+
+import styles from './styles.module.scss';
+
+class MatchHistory extends Component {
+  state = {
+    matches: []
+  }
+
+  componentDidMount() {
+    services.getMatches().then(response => this.setState({ matches: response.data }));
```

Comment:
Create a function that checks the loading state, the error state and the matches